platform/mellanox/mlnx-platform-api/sonic_platform/sfp.py

```diff
@@ -63,6 +63,24 @@
 XCVR_VENDOR_DATE_WIDTH = 8
 XCVR_DOM_CAPABILITY_OFFSET = 92
 XCVR_DOM_CAPABILITY_WIDTH = 2
+# to improve performance we retrieve all eeprom data via a single ethtool command
+# in function get_transceiver_info and get_transceiver_bulk_status
+# XCVR_INTERFACE_DATA_SIZE stands for the max size to be read
+# this variable is only used by get_transceiver_info.
+# be attention that each time some new value added to the function 
```

Comment:
change "be attention" to "please be noted" ?

Reply:
fixed.